frontend/tests/unit/services/transactions/utils.spec.ts

```diff
@@ -2,18 +2,88 @@ import * as providers from '@ethersproject/providers';
 import * as ethers from 'ethers';
 
 import {
+  CONFIRMATION_TIME_BLOCKS,
   getCurrentBlockNumber,
   getJsonRpcProvider,
   getLatestBlock,
   getReadOnlyContract,
   getReadWriteContract,
+  getSafeEventHandler,
 } from '@/services/transactions/utils';
 import { getRandomEthereumAddress, getRandomUrl } from '~/utils/data_generators';
+import { MockedEvent, MockedTransactionReceipt } from '~/utils/mocks/ethers';
 
 vi.mock('@ethersproject/providers');
 vi.mock('ethers');
 
 describe('utils', () => {
+  describe('getSafeEventHandler()', () => {
+    it('returns a re-org safe version of the passed handler', () => {
```

Comment:
Nitpick: I think this test name is a bit far-fetched. It just ensures that a function is returned.

Reply:
True ✅